README.md

```diff
@@ -2,9 +2,36 @@
 
 geOrchestra Maelstro is an application which helps synchronise geonetwork and geoserver instances
 
-## Quick start with geOrchestra
+## Docker deploy
 
-Before starting developement you need to setup some geOrchestra configurations.
+### Solo quick start (without included geOrchestra composition)
+
+Maelstro can be used outside geOrchestra.
+
+First select the `docker-compose-solo.yml` file as current composition:
+
+```bash
+ln -s docker-compose-solo.yml docker-compose.yml
```

Comment:
why not choose docker-compose-geOrchestra.yml or docker-compose-solo.yml as default (docker-compose-solo.yml), which can be launched via docker `compose up -d` ?
The other version can be launched via `docker compose -f docker-compose-solo.yml up -d` This is a more standard way for docker compos, I find the symlinks quite confusing.

However this is a minor issue, the app is working well.